### PR TITLE
UX: update label from reply to answer on Q&A topics

### PR DESCRIPTION
**Pull request by techAPJ**
Before:

Screen Shot 2022-06-02 at 15 38 03

After:

Screen Shot 2022-06-02 at 15 34 58

### CODE REVIEW

File: assets/javascripts/discourse/initializers/extend-composer-actions.js [hunk 1]
```diff
@@ -16,11 +16,23 @@ export default {
 
       api.customizeComposerText({
         actionTitle(model) {
-          return model.createAsQA ? I18n.t("composer.create_qa.label") : null;
+          if (model.createAsQA) {
+            return I18n.t("composer.create_qa.label");
+          } else if (model.topic && model.topic.is_qa) {
```
Comment (CvX) [inline assets/javascripts/discourse/initializers/extend-composer-actions.js:21]:
```suggestion
          } else if (model.topic?.is_qa) {
```

File: assets/javascripts/discourse/initializers/extend-composer-actions.js [hunk 2]
```diff
@@ -16,11 +16,23 @@ export default {
...
+          } else {
+            return null;
+          }
         },
 
         saveLabel(model) {
-          return model.createAsQA ? "composer.create_qa.label" : null;
+          if (model.createAsQA) {
+            return "composer.create_qa.label";
+          } else if (model.topic && model.topic.is_qa) {
```
Comment (CvX) [inline assets/javascripts/discourse/initializers/extend-composer-actions.js:31]: JarekLint 😂 

```suggestion
          } else if (model.topic?.is_qa) {
```

**Review by CvX — APPROVED** [review]

File: assets/javascripts/discourse/initializers/extend-composer-actions.js (hunk 2 above)
Comment (techAPJ) [reply assets/javascripts/discourse/initializers/extend-composer-actions.js]: Updated! Thanks for the suggestion. 👍🏼

**Comment by techAPJ** [comment]
Thanks for the review @CvX ❤️

**Pull request merged**